backend/debug_transaction_descriptions.py

```
@@ -0,0 +1,174 @@
+This helps understand why the SAINSBURYS pattern didn't match any transactions.
+"""
+
+import os
+import sys
+import logging
+from typing import List, Dict
+
+# Add src directory to path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'src'))
+
+from utils.db_utils import list_user_transactions
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)